src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -171,15 +171,11 @@ public void jobLogic() {
                 status.fail("Could not clone version.", e);
             }
         }
-
         // FIXME: Should we overwrite the input GTFS dataset if transforming in place?
```

Comment:
It now appears like this FIXME is associated with the auto deploy, which is a bit confusing.

Reply:
@landonreed can this be addressed as part of this PR? Looking at the dev branch this FIXME doesn't seem to be associated with anything in particular.

Reply:
Yea, I think this FIXME can actually be removed.